app/services/idv/session.rb

```diff
@@ -16,7 +16,6 @@ class Session
       personal_key
       personal_key_acknowledged
       phone_for_mobile_flow
-      phone_with_camera
```

Comment:
This session attribute removal should be okay to deploy at the same time as deleting the controller that uses it, because the feature flag has been turned off for a while.